Makefile

```diff
@@ -9,21 +9,28 @@ install:
 		make _get_gcp_credentials; \
 	fi
 	make _initiate_env
-	pyenv install 3.10.4 -s
+	# Install UV and create the venv
```

Comment:
pareil que ci-dessous, peux tu rajouter la ligne 
̀uv || echo -e "\n \n Please restart you shell in order to allow uv commands \n \n"`

avant le `uv venv... `